src/components/Api.js

```diff
@@ -0,0 +1,142 @@
+class Api {
+  constructor({ baseUrl }) {
+    this._baseUrl = baseUrl;
+    this._headers = {
+      authorization: "6d7ae9eb-8869-410f-b35c-8e1dcc44f04b",
+      "Content-Type": "application/json",
+    };
+  }
+  //GET https://around-api.en.tripleten-services.com/v1/cards
+  getInitialCards() {
+    return fetch(`${this._baseUrl}/cards`, {
+      method: "GET",
+      headers: this._headers,
+    })
+      .then((res) => {
```

Comment:
Needs correction: The code for processing server response gets duplicated. You should declare a method and reuse it

```js
_processResponse = (res) => { 
      if (res.ok) { 
        return res.json(); 
      } 
      return Promise.reject(`Error: ${res.status}`); 
}

getInitialCards() { 
  ...
  .then(this._processResponse)
}
```